content/support-training.html

```diff
@@ -49,6 +49,12 @@ <h2>Courses</h2>
               <img src="../assets/commercial-support-logos/robust-perception.png" alt="Robust Perception">
             </a>
           </div>
+
+          <div class="col-md-3 col-sm-6 doc-content tc-col">
+            <a href="https://www.it-schulungen.com/seminare/netzwerktechnologien/prometheus/index.html" class="btn btn-training btn-default btn-large" target="_blank">
+              <img src="../assets/commercial-support-logos/it-schulungen-com.png" alt="Prometheus Schulungen bei IT-Schulungen.com">
```

Comment:
The image link is broken because it contains `-com` in the image file name, while the actual image file doesn't have that naming component.